components/scream/scripts/test_all_scream.py

```diff
@@ -526,8 +528,9 @@ def generate_ctest_config(self, cmake_config, extra_configs, test):
         test_dir = self.get_test_dir(self._work_dir,test)
         num_test_res = self.create_ctest_resource_file(test,test_dir)
         cmake_config += " -DSCREAM_TEST_MAX_TOTAL_THREADS={}".format(num_test_res)
+        verbosity = "-V --output-on-failure" if not self._extra_verbose else "-VV"
```

Comment:
I know this was already the case in master, but why do we need `-V` when we already have `--output-on-failure`? On my laptop, `-V --output-on-failure` is the same as `-V`, with all test output being printed. Also, I don't get any difference between `-V` and `-VV` on my laptop. CTest docs are not very clear on which "extra" output should be shown by `-VV` that is not shown by `-V`.

Can we just do `verbosity = "-V" if self._extra_verbose else "--output-on-failure"`?